android/src/main/java/com/thebluealliance/androidclient/helpers/MatchHelper.java

```diff
@@ -593,14 +609,24 @@ public static String getMatchTitleFromMatchKey(String matchKey) {
             }
 
             if(set == null) {
-                // No set specified; this is a quals match
-                return LONG_TYPES.get(TYPE.fromShortType(type)) + " " + number;
+                // No set specified; this is a match like "Quals 10" (abbrev "Q10")
+                String format = abbrev ? "%1$s%2$s" : "%1$s %2$s";
```

Comment:
You should just be able to make it a method parameter.

BroadcastReceiver's [onReceive](https://developer.android.com/reference/android/content/BroadcastReceiver.html#onReceive%28android.content.Context,%20android.content.Intent%29) has the current context as a parameter, so it'll be available.